### ISSUE TITLE
[FeatureRequest] Dropdown/Toggle for request variants

### ISSUE THREAD

**Issue by AskMeAgain** [a85/POSTMan-Chrome-Extension#10048]
**Is your feature request related to a problem? Please describe.**
Currently we have a request which sends "data1" to a rest endpoint. Sometimes we need to send "data2" to this endpoint for testing purposes. After this request, we issue multiple other requests automated.

**Describe the solution you'd like**
A simple dropdown, for each request, with prefilled data, would allow us (and our project managers), to issue request "variants" on a button click.

We can now have a single Request, and the dropdown/toggle lets us select specific request variants, like sending working or non-working data for this request.

Bonus points if we can have multiple dropdowns per request.

**Describe alternatives you've considered**
Yes we could create a complete new request which does send non-working data, but this would mean that we need to duplicate the request just for a different value. If we now update the api, we need to put the changes in each variant of the request. A simple dropdown selection would work wonders here.

**Api usage**
Api usage for requests would look like this: 

    postman.requestDropdown.getValue();

or directly in the request body:

    {{$dropdownValue}}

**Comment by giridharvc7** [comment]
Hey, have you tried using Environments and variables for this?

**Comment by AskMeAgain** [comment]
yes, but this is just not scalable, as the moment you have more then 4 different requests, each with 2 variants, would result in 2⁴ environments. Not even considering "real" environments like int, sandbox, develop etc. This would just change the number of environments to 2⁴ * 3, just for this basic thing.

Also using variables here (that is what we are doing currently) is just not really user friendly as our pms now need to edit request bodies so they do what they want.

It would just be much more convenient if we could get an additional dropdown but only local to a request.
